lib/game.rb

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class Game
+def initialize
+  @new_board = Board.new
```

Comment:
Per good OOP practice, we should not instantiate a class inside another class. In this case, we are making `Game` class too dependent on the `Board` and `Player` classes. If there are changes we need to make into these two other classes (which we do) that would mean there are changes we need to make in our `Game` class. For instance, what if now we want `Player.new` to take in arguments? Then we need to adjust lines 9 and 10 to accommodate this change to `Player` also.

Good OOP design is that we want `Game` to not care what's happening in the other two classes. All `Game` should know is the public methods of those classes that it should call to manage the flow of the game.

So with this we just need to pass instantiated `Player` and `Board` objects as arguments to `Game.new`. We then need to instantiate `Player` and `Board` outside of `Game` (i.e, in `main.rb`).